forem/forem
# Fix flag user username text interpolation

## What type of PR is this? (check all applicable)


## Description

Fixes a small bug in the "moderate" section for comments

## QA Instructions, Screenshots, Recordings

1. Boot app locally
2. Open a Post with comments
3. click on comment options dropdown and "moderate"
4. Ensure the text interpolation works as expected

**Before**

<img width="1352" alt="Screen Shot 2022-10-18 at 11 42 10" src="https://user-images.githubusercontent.com/6045239/196507082-9d940286-8e9a-438a-b1de-b11a38457d4d.png">

**After**

<img width="1332" alt="Screen Shot 2022-10-18 at 11 47 06" src="https://user-images.githubusercontent.com/6045239/196507111-10cba587-d585-4234-9098-ecb9129ded66.png">

### UI accessibility concerns?

N/A

## Added/updated tests?

- [x] No, and this is why: There were no errors in place except for a missing text interpolation and we don't have tests for these types of typos

## [Forem core team only] How will this change be communicated?

- [x] This change does not need to be communicated, and this is why not: Small bug fix

## [optional] What gif best describes this PR or how it makes you feel?

![Nicholas Cage: I don't know why, you tell me why](https://media.giphy.com/media/BI9cy6CIW4JwydMbkk/giphy.gif)
